db/seeds.rb

```diff
@@ -77,6 +77,10 @@
 family = Family.create!(male: male, female: female, organization_id: white_abalone.id)
 tank = Tank.create!(facility: Facility.find_by(code: 'PSRF'), name: 'AB-17', organization_id: white_abalone.id)
 Operation.create!(tank: tank, animals_added: 800, family: family, operation_date: 7.days.ago, action: :add_family, organization_id: white_abalone.id)
-measurement_event = MeasurementEvent.create!(name: "My first measurement", tank: tank, organization_id: white_abalone.id)
-Measurement.create!(name: 'count', value_type: 'integer', value: '743', measurement_event: measurement_event, date: 3.days.ago, organization_id: white_abalone.id)
-Measurement.create!(name: 'count', value_type: 'integer', value: '719', measurement_event: measurement_event, date: 1.days.ago, organization_id: white_abalone.id)
+measurement_event = MeasurementEvent.create!(name: "My first measurement", organization_id: white_abalone.id)
+measurement_type = MeasurementType.create!(name: "length", unit: "cm", organization: white_abalone)
+measurement_type = MeasurementType.create!(name: "count", unit: "number", organization: white_abalone)
+measurement_type = MeasurementType.create!(name: "weight", unit: "g", organization: white_abalone)
+measurement_type = MeasurementType.create!(name: "gonad score", unit: "number", organization: white_abalone)
+Measurement.create!(value: '743', measurement_event: measurement_event, date: 3.days.ago, organization_id: white_abalone.id, subject: male, measurement_type: measurement_type)
+Measurement.create!(value: '719', measurement_event: measurement_event, date: 1.days.ago, organization_id: white_abalone.id, subject: female, measurement_type: measurement_type)
```

Comment:
I think this is fine, but we will probably need to revisit these seeds at some point in the future to make sure that it sets up the application  in a state that accurately portrays its daily usage.

Reply:
I agree. I've felt like we have slightly been writing these tests to pass from a function perspective but not necessarily use case perspective. Shall we add a backlog task to review the seeds file and make some amendments where seen fit? For instance I think both organizations should have measurements, not just white_abalone like this.